.pubnub.yml

```diff
@@ -0,0 +1,115 @@
+name: kotlin
+version: 1.0.0
+schema: 1
+scm: github.com/pubnub/kotlin
+files:
+  - build/libs/pubnub-kotlin-1.0.0-all.jar
```

Comment:
Please change `build/libs/pubnub-kotlin-1.0.0-all.jar` to `build/libs/pubnub-kotlin-1.0.1-all.jar` to point to the latest jar.

Reply:
The version is fine. This is not 1.0.1 yet.